backend/api/Pims.Api.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <UserSecretsId>0ef6255f-9ea0-49ec-8c65-c172304b4926</UserSecretsId>
-    <Version>1.1.0-16.32</Version>
+    <Version>1.1.0-16.33</Version>
```

Comment:
not a big deal - but shouldn't we be incrementing the hotfix version (not the build version) for every hotfix applied to master?

Reply:
Right, will do on future PRs.